config/migrations/20221013105306-blocked-mock-users.ttl

```diff
@@ -0,0 +1,65 @@
+@prefix mu: <http://mu.semte.ch/vocabularies/core/> .
```

Comment:
Deze test-data zou ik vermijden om in de git repos te steken, want die gaat ook op productie terecht komen.